### DISCUSSION TITLE
Github dark theme not working on iOS 15 Safari

### DISCUSSION THREAD

**Discussion by Seterplus** [community/community#125101] [Mobile]
### Select Topic Area

Bug

### Body

The dark mode makes the whole page pure black on iOS 15 Safari. This is believed to be a bug for dark mode. There are discussions on [Reddit](https://www.reddit.com/r/github/comments/1bxqkef/has_github_changed_anything_recently_ios_152/) but I don't find any topics here. Screenshots from that post:

![image](https://github.com/community/community/assets/6801579/ad3c82d5-a127-4931-83c1-daa80fda0a05)

**Comment by Janethq** [comment]
Changing the default Safari experimental settings for VP9 decoding seems to resolve the issue. Specifically, enabling "VP9 SW Decoder on Battery" and disabling "VP9 Decoder" has been found to work for some users.

Otherwise, disabling "WebRTC Platform UDP Sockets" in the iOS Safari Advanced Experimental Features has also been found to work.

**Reply by Seterplus** [reply]
Hmmm, just realized my iOS is 14.8 and I don't have these exact settings. However, enabling "WebRTC VP9 profile 0 codec"/"WebRTC VP9 profile 2 codec" and disabling "WebRTC Platform Sockets" doesn't work for me - actually I switched every single setting and it still didn't work.

**Comment by PoomSmart** [comment]
Their new way of applying CSS on dark mode is not supported on iOS <= 15.3

https://x.com/PoomSmart/status/1779922660220436776
